app/Models/Camp.php

```diff
@@ -193,4 +195,153 @@ public static function getAllParentCamp($topicNum, $filter, $asOfDate = null)
         ->whereRaw('go_live_time in (select max(go_live_time) from camp where topic_num=' . $topicNum . ' and objector_nick_id is null and go_live_time < '.$asOfDate.' group by camp_num)')
         ->orderBy('submit_time', 'desc')->orderBy('camp_name','desc')->groupBy('camp_num')->get();
     }
+
+    public static function getAllChildCamps($camp) {
```

Comment:
Can you please add :array() ?
And also take the `{` to the next line. Just to be consistent